online-resources/learn-to-code-resources.md

```diff
@@ -164,8 +164,12 @@
 
 * [You Learn Haskell for Great Good](http://learnyouahaskell.com/chapters)
 
-
-##Free Lynda Account
+## Free Lynda Account
```

Comment:
This is a drive-by, but whilst I was here I thought I'd fix the markdown on the entry above 😄

Reply:
Thank you! Most appreciated 💃